addon/README.md

```diff
@@ -16,7 +16,7 @@ dev: [dev-prefs.json](dev-prefs.json)
 
 You may need to set both `xpinstall.signatures.required` and
 `xpinstall.whitelist.required` flags to false in `about:config`
-if the addon has not yet been signed.
+if the add-on has not yet been signed.
```

Comment:
Still relevant for local development - we have to bump the version each and every time we want a new signed XPI, so that's totally unworkable for local dev.

So, we use Fx Dev Edition, turn off sig checking, and run `jpm watchpost` via `npm run watch` along with the [Extension Autoinstaller](https://addons.mozilla.org/en-US/firefox/addon/autoinstaller/).